user_groups/user_groups.md

```diff
@@ -6,7 +6,8 @@
 [Add a user to a user group](#add-a-user-to-a-user-group) <br />
 [Remove a user from a user group](#remove-a-user-from-a-user-group) <br />
 [Get user group permissions](#get-user-group-permissions) <br />
-[Add or update user group permission for a given resource](#add-or-update-user-group-permission-for-a-given-resource) <br />
```

Comment:
addin and updating should have its own section, fundamentally they are two different apis

Reply:
yep they are in two different sections @brennyjun